tests/test1.in

```diff
@@ -33,12 +33,12 @@ end
 # Simple alias tests
 
 function foo
-    echo >foo.txt $argv
+    echo >/tmp/fish_foo.txt $argv
```

Comment:
That's a problem if you do `make install` then `make DESTDIR=/path/to/install/prefix/ test` since that makes *test_root* a symlink outside the git repo. The other option is to create a *test_temp* directory and use that. Which has the advantage of not having to clean up individual files left in */tmp* by the tests. Yes? No?